editor/src/components/inspector/add-remove-layout-system-control.tsx

```diff
@@ -99,26 +89,44 @@ export const AddRemoveLayoutSystemControl = React.memo<AddRemoveLayoutSystemCont
 
   const addLayoutSystemOpenerButton = React.useCallback(
     () => (
-      <SquareButton
-        data-testid={AddRemoveLayoutSystemControlTestId()}
-        highlight
-        onClick={NO_OP}
-        style={{ width: 12, height: 22 }}
-      >
-        <Icn category='semantic' type='plus' width={12} height={12} />
+      <SquareButton highlight onClick={NO_OP} style={{ width: 12, height: 22 }}>
+        <Icons.Threedots />
       </SquareButton>
     ),
     [],
   )
 
   const addLayoutSystemMenuDropdownItems = React.useMemo(
-    (): DropdownMenuItem[] => [
-      { id: AddFlexLayoutOptionId, label: 'Flex', onSelect: addFlexLayoutSystem },
-      { id: AddGridLayoutOptionId, label: 'Grid', onSelect: addGridLayoutSystem },
-    ],
-    [addFlexLayoutSystem, addGridLayoutSystem],
+    (): DropdownMenuItem[] =>
+      stripNulls([
+        { id: AddFlexLayoutOptionId, label: 'Flex', onSelect: addFlexLayoutSystem },
+        { id: AddGridLayoutOptionId, label: 'Grid', onSelect: addGridLayoutSystem },
+        optionalMap(
+          () => ({
+            id: RemoveLayoutSystemOptionId,
+            label: 'Remove',
+            onSelect: removeLayoutSystem,
+            danger: true,
+          }),
+          layoutSystem,
+        ),
+      ]),
+    [addFlexLayoutSystem, addGridLayoutSystem, layoutSystem, removeLayoutSystem],
   )
 
+  const label = () => {
```

Comment:
If you did this just to use a switch case without mutation, you could still just call it immediately:

```typescript

const label = (() => {
    switch (layoutSystem) {
      case null:
        return 'Layout System'
      case 'flex':
        return 'Flex'
      case 'grid':
        return 'Grid'
      default:
        assertNever(layoutSystem)
    }
  })()
```

Reply:
Or just use `React.useMemo`

Reply:
I didn't want to make it a const because afaik we don't like chained ternary expressions, I didn't want to use a useMemo because it's too heavyweight and unnecessary because the end product is a string. As far as I understand the IIFE is the same as making this a function and calling it in the returned JSX, without the syntactic noise of the IIFE